dev/container/Dockerfile

```diff
@@ -1,20 +1,17 @@
-ARG RUBY_VERSION
-FROM ruby:$RUBY_VERSION-buster
+ARG RUBY_VERSION=3.2
+FROM ruby:$RUBY_VERSION-alpine as builder
 
-# Install dependencies
-RUN \
-  apt-get update && \
-  apt-get install --yes libffi-dev
-
-# Show full path in prompt
-RUN echo 'PS1='\''imap-backup:$(pwd)>'\''' > /etc/bash.bashrc
-
-# Create binstubs (including one for imap-backup) so we can run it
-# without using `bundle exec`
-ENV PATH ./bin/stubs:$PATH
+# install dependencies
+RUN apk add libffi-dev alpine-sdk
+RUN gem install bundler --version=2.3.22
+COPY . .
+RUN bundle install
 
+FROM ruby:$RUBY_VERSION-alpine as final
+ENV HOME=/config \
```

Comment:
As noted elsewhere, relying on the presence of a `config` directory may be problematic. Especially when using the image for development.

Reply:
omitting HOME causes the program to break, reverted to `/app`